### ISSUE TITLE
Integration into another distro

### ISSUE THREAD

**Issue by blackstarek** [pimusicbox/pimusicbox#334]
Hey , is there a possibility of integrating musicbox into another distro on RPI?

The main reason why I'm asking this question is, that im using RetroPie as a Distro and I dont want to swap SD-Cards everytime.

**Comment by kingosticks** [comment]
Is that based on Debian? There's already an issue to create a debian
package and it is the long term goal.

**Comment by blackstarek** [comment]
Yes, it is based on debian :] So before that happens, I guess i should just install mopidy right?
